alf/environments/alf_wrappers_test.py

```diff
@@ -170,5 +172,78 @@ def test_duration_applied_after_episode_terminates_early(self):
         self.assertTrue(last_time_step.is_last())
 
 
+class MultitaskWrapperTest(alf.test.TestCase):
+    def test_multitask_wrapper(self):
+        env = alf_wrappers.MultitaskWrapper.load(
+            suite_gym.load, ['CartPole-v1', 'CartPole-v1'])
```

Comment:
Should check task names to be different because they are used as keys for env_info

Reply:
added